.github/workflows/trigger_doc_build.yml

```diff
@@ -0,0 +1,18 @@
+name: Trigger documentation build on Readthedocs
+
+on:
+  schedule:
+    # Run this action twice a month just before midnight UTC
+    # "*" is a special character in YAML, so we have to quote this string
+    - cron:  '59 23 7,22 1-12 *'
+  workflow_dispatch:
+
+jobs:
+  trigger_readthedocs_build:
+    runs-on: ubuntu-18.04
+
+    steps:
+      - run: |
+          curl -X POST \
+            -d "token=${{ secrets.RTD_WEBHOOK_TOKEN }}" \
+            "${{ secrets.RTD_WEBHOOK_URL }}"
```

Comment:
This command is from here: https://docs.readthedocs.io/en/stable/integrations.html#parameters